tests/unit/test_credentials.py

```diff
@@ -0,0 +1,49 @@
+import mock
+import pytest
+from bitbucket.conf import settings
+from bitbucket.credentials import KatkaCredentialsService
+from bitbucket.exceptions import BitbucketBaseAPIException
+from requests import HTTPError
+from rest_framework.exceptions import PermissionDenied
+
+
+@mock.patch('bitbucket.base.KatkaService.get')
+class TestKatkaCredentialsService:
+    def test_get_access_token(self, mock_katka_service_get):
+        response = mock.Mock()
+        response.json.return_value = {
+            'credential_public_identifier': 'wonder_women',
+            'secret_public_identifier': 'jean_grey',
```

Comment:
I wonder if we should use UUIDs here, as the public identifiers are

Reply:
ok, changed it to make it more clear.